bitnami/rabbitmq-cluster-operator/templates/cluster-operator/deployment.yaml

```
@@ -88,6 +88,10 @@ spec:
           {{- else }}
           args:
             - --metrics-bind-address=:{{ .Values.clusterOperator.containerPorts.metrics }}
+            - --zap-devel={{ with .Values.clusterOperator.logging }}{{ default "false" .development }}{{ else }}"false"{{ end }}
+            - --zap-encoder={{ with .Values.clusterOperator.logging }}{{ default "json" .encoder }}{{ else }}"json"{{ end }}
+            - --zap-log-level={{ with .Values.clusterOperator.logging }}{{ default "error" .logLevel }}{{ else }}"error"{{ end }}
+            - --zap-stacktrace-level={{ with .Values.clusterOperator.logging }}{{ default "error" .stacktraceLevel }}{{ else }}"error"{{ end }}
```

I think this logic can be simplified, I think all these condition only makes sense when an user remove the logging configuration with `--set clusterOperator.logging=null`. In that case I think the following could be applied (because the user wants to remove the logging section):
```suggestion
            {{- if .Values.clusterOperator.logging }}
            - --zap-devel={{ .Values.clusterOperator.logging.development }}
            - --zap-encoder={{ .Values.clusterOperator.logging.encoder }}
            - --zap-log-level={{ .Values.clusterOperator.logging.logLevel }}
            - --zap-stacktrace-level={{ .Values.clusterOperator.logging.stacktraceLevel }}
            {{- end }}
```
I think this is little bit clearer than the initial approach. WDYT?